dist/Locale-Maketext/lib/Locale/Maketext.pm

```diff
@@ -136,19 +136,21 @@ sub fail_with { # an actual attribute method!
 
 #--------------------------------------------------------------------------
 
-sub blacklist {
-    my ( $handle, @methods ) = @_;
+sub _exclude {
+    my ( $caller, $handle, @methods  ) = @_;
```

Comment:
Why `$caller`? It appears not to be used? Is it related to an earlier version of `$name`?

Reply:
Caller isn’t used. I will remove it.

Reply:
updated code based on @Leont 's request